### PR TITLE
Fixed issues on Python CLI dump issues:

### PR DESCRIPTION
**Pull request by weiqiand**
1. Initialized the flags handle.
2. make sure the flag parameter passing at right position parameters.

### CODE REVIEW

File: tdi_python/tdicli.py [hunk 1]
```diff
@@ -62,83 +62,83 @@ def tdi_table_entry_get(tbl_hdl, session, dev_tgt, key, data, flag):
...
         def tdi_table_entry_key_get(tbl_hdl, session, dev_tgt, handle, e_tgt, key):
-            flags = c_uint64(0)
+            flags = self.get_flags()
             return self._driver.tdi_table_entry_key_get(tbl_hdl, session, dev_tgt, flags, handle, e_tgt, key)
         setattr(self, 'tdi_table_entry_key_get', tdi_table_entry_key_get)
 
         def tdi_table_entry_get_first(tbl_hdl, session, dev_tgt, key, data, flag):
-            flags = c_uint64(flag.value)
+            flags = self.get_flags()
+            print("tdi_table_entry_get_first data={} byref={} key={}".format(str(data), str(byref(data)), str(key)))
```
Comment (saynb) [inline tdi_python/tdicli.py:81]: remove unnecessary logs ?

File: tdi_python/tdicli.py [hunk 2]
```diff
@@ -62,83 +62,83 @@ def tdi_table_entry_get(tbl_hdl, session, dev_tgt, key, data, flag):
         setattr(self, 'tdi_table_entry_get', tdi_table_entry_get)
 
         def tdi_table_entry_get_by_handle(tbl_hdl, session, dev_tgt, ent_hdl, key, data, flag):
-            flags = c_uint64(flag)
+            flags = self.get_flags()
```
Comment (saynb) [inline tdi_python/tdicli.py:65]: How are we converting the flag input by user to the flags object? Shouldn't we be converting that? This doesn't look correct

File: tdi_python/tdicli.py (hunk 1 above)
Comment (weiqiand) [reply tdi_python/tdicli.py:81]: yes. done

File: tdi_python/tdicli.py (hunk 2 above)
Comment (weiqiand) [reply tdi_python/tdicli.py:65]: get_flags() is function to access previous created self._driver.tdi_flags_create(flag,  byref(self._flags)) and saved on  self._flags

**Review by saynb — APPROVED** [review]

**Comment by weiqiand** [comment]
Open an enhancement tracking for Flags object (allow user to set/mod for each associated APIs).

**Pull request merged**